WordPress/openverse
# Treat 403s from Europeana as dead links

## Fixes



Fixes #2417 by @krysal

## Description


Adds the exception for Europeana as said in the issue.

## Testing Instructions

1. To manually test these changes, copy this row to the **sample_data/sample_image.csv** file
```
27d7186f-2f8e-4d24-badd-d6e4bdc60071,2022-11-27 14:37:37.668997+00,2022-11-27 14:37:37.668997+00,provider_api,europeana,europeana,/2022709/oai_fototeca_mcu_es_fototeca_MORENO_04766_C,https://www.mcu.es/fototeca_patrimonio/show_ficha.do?archivo=MORENO&record=04766_C,https://www.mcu.es/fototeca_patrimonio/Visor?usarVisorMCU=false&archivo=MORENO/preview/04766_C_P.jpg,,,,,by-nc-nd,4.0,,,Arco de flechas,"{""license_url"": ""https://creativecommons.org/licenses/by-nc-nd/4.0/"", ""raw_license_url"": null}",,f,2022-11-27 14:37:37.668997+00,f,jpg,,
```
2. Confirm you have the environment variable `FILTER_DEAD_LINKS_BY_DEFAULT` set to `True` you don't have it set to anything (it's true by default)
3. Run `just api/recreate`
4. Confirm the following search returns zero result http://localhost:50280/v1/images/?source=europeana

## Checklist

- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
